src/components/SightingDetail/SightingDetail.tsx

```diff
@@ -1,37 +1,35 @@
 import "./SightingDetail.scss"
 import React, { useEffect, useState } from 'react';
-import { fetchSightingById, WhaleSighting} from "../../clients/apiClient";
+import { fetchSightingById, WhaleSighting } from "../../clients/apiClient";
 import "./SightingDetail.scss";
 
 interface SightingDetailProps {
     sightingId: number;
 }
-export function SightingDetail(props: SightingDetailProps): JSX.Element {
-    const [sighting, setSighting] = useState<any>(null);
+export function SightingDetail({ sightingId }: SightingDetailProps): JSX.Element {
+    const [sighting, setSighting] = useState<any>();
```

Comment:
I think you might have forgotten to change the type back to `<WhaleSighting>` now we've updated the interface to have lowercase properties?

Reply:
changed